impl/c-qube/src/services/csv-adapter/parser/utils/csvreader.ts

```diff
@@ -1,15 +1,15 @@
 const fs1 = require('fs');
 const fs = require('fs').promises;
-
 import * as csv from 'csv-parser';
 
 export async function readCSV(filePath: string): Promise<string[][]> {
   return new Promise((resolve, reject) => {
     const rows: string[][] = [];
-    // TODO: Add checking here
+    const configPath = 'ingest/config.json'; // Update the path to your config.json file
```

Comment:
don't hardcode the config path here, either take the configPath as a function argument or the delimiter character, incase neither is present set default to `,`, and in case both are present always give preference to reading the config. After making this change ideally edit the core top level services whereever this function is being called to set the delimiter.